source/common/router/router.h

```diff
@@ -471,6 +473,7 @@ class Filter : Logger::Loggable<Logger::Id::router>,
   bool downstreamEndStream() const override { return downstream_end_stream_; }
   uint32_t attemptCount() const override { return attempt_count_; }
   const std::list<UpstreamRequestPtr>& upstreamRequests() const { return upstream_requests_; }
+  void onUpstreamConnectionEstablished() override;
```

Comment:
If this is from the new interface please comment so.

Reply:
just to be sure do you want me to comment in the code that this is coming from the router interface or comment in this PR ?